src/opsignupclasses.py

```diff
@@ -147,7 +147,10 @@ async def send_message(self,ctx,date):
             pass
 
         for reaction in self.reactions.keys():
-            embed.add_field( name = f'{self.reactions[reaction].symbol} {self.reactions[reaction].name}', value = f'{self.reactions[reaction].members["perm"]}', inline=True)
+			if self.reactions[reaction].maxReact > 0:
+				embed.add_field( name = f'{self.reactions[reaction].symbol} {self.reactions[reaction].name}', value = f'LIMIT: {self.reactions[reaction].maxReact}\n{self.reactions[reaction].members["perm"]}', inline=True)
+			else:
+				embed.add_field( name = f'{self.reactions[reaction].symbol} {self.reactions[reaction].name}', value = f'{self.reactions[reaction].members["perm"]}', inline=True)
```

Comment:
Match the formatting of the system with and without a signup.

i.e 0/2 instead of 2